frontend/src/components/PageHeader/PageHeader.tsx

```diff
@@ -2,7 +2,7 @@ import React, { ReactNode } from 'react';
 import styles from './PageHeader.module.css';
 
 export interface PageHeaderProps {
-  name: string,
+  name?: string,
   children?: ReactNode
```

Comment:
React.FC already contains optional prop 'children', so it not necessary here

Reply:
right, done 👍